src/clojure_tictactoe/game/game_play.clj

```diff
@@ -8,27 +8,36 @@
             [clojure-tictactoe.game.game-rules :as rules]
             ))
 
-(defn player-move [move-function board]
-  (board/mark-space (move-function board) @players/player-atom board))
+(def standard-board-size 9)
+(def game-options ["Human vs Human" "Human vs. Computer"])
+(def players '("player 1" "player 2"))
```

Comment:
Is there a reason these are used as `def`'s and not `let` bindings?

Reply:
I guess there is not?